src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -413,9 +410,6 @@ const AppWrapper = (props: { children: ReactNode }) => {
     Agent.agent.connections.onConnectionStateChanged((event) => {
       return connectionStateChangedHandler(event, dispatch);
     });
-    Agent.agent.signifyNotifications.onNotificationStateChanged((event) => {
-      return keriaNotificationsChangeHandler(event, dispatch);
```

Comment:
OK, Robert and I have agreed that the best thing to do is call `getMultisigIcpDetails` when the user clicks to open the multi-sig notification in the UI.

If `getMultisigIcpDetails` throws an error with `(error as Error).message == MultiSigService.UNKNOWN_AIDS_IN_MULTISIG_ICP`, then we will provide feedback to the user that they haven't finished scanning all of the OOBIs. I'm not sure yet when this design will be ready - if we don't do it in your PR @sdisalvo-crd we can create another ticket as follow up for this error handling.

@Sotatek-HocNguyena This means we won't need to poll `getMultisigIcpDetails` in the background, which is great. :)

Reply:
@sdisalvo-crd Here is what Rob is proposing: https://www.figma.com/design/XN6K76yt13zxCUHPh1Egr8/Identity-wallet?node-id=10451-17217&t=z9dZWue9cWBDjNw6-1

Since it's a full page, I guess we should capture this in another ticket and do it after? (are refine if necessary too anyway)

Reply:
Ticket created for refinement: https://cardanofoundation.atlassian.net/browse/DTIS-975